src/TestSuite/Fixture/FixtureManager.php

```diff
@@ -352,11 +376,17 @@ public function loadSingle($name, $db = null, $dropTables = true)
                 $db = ConnectionManager::get($fixture->connection());
             }
 
+            $fixture->dropConstraints($db);
```

Comment:
Ah, I mis-read your comment.
Yes it is needed given that constraints are stored separately when the fixture schema is built from the ``$fields`` property : https://github.com/HavokInspiration/cakephp/blob/fixtures-order/src/TestSuite/Fixture/TestFixture.php#L171-L175